exercises/queen-attack/description.md

```diff
@@ -22,4 +23,36 @@ So if you are told the white queen is at `c5` (zero-indexed at column 2, row 3)
 ```
 
 You are also be able to answer whether the queens can attack each other.
-In this case, that answer would be yes, they can, because both pieces share a diagonal.
+In this case, that answer would be yes; they can, because both pieces share a diagonal.
+
+### Examples of queens attacking:
+
+```text
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ W _ _ _ _
+_ _ _ W _ _ _ _     _ B _ _ _ W _ _     _ _ _ _ _ _ _ _
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _
+_ _ _ _ _ _ B _     _ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ B _ _ _ _
+_ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _     _ _ _ _ _ _ _ _
+```
```

Comment:
This could use an explanation of what you're seeing.

Reply:
An example of queens attacking already exists in the description. I am not convinced that we need more. The textual description `In the game of chess, a queen can attack pieces which are on the same row, column, or diagonal.` sounds clear enough to me.

However, if we do want more examples, then:
- They should all use the same notation, the existing ones have labels "a b c..." and "1 2 3...".
- They should be grouped together, not scattered in two different places.
- +1 to @SleeplessByte that each example should come with an explanation.

Reply:
I'm also of the opinion that these examples are not needed. I do like the explaining of B and W.